fsharp-backend/tests/testfiles/httpclient.tests

```diff
@@ -30,23 +30,53 @@ HttpClient.basicAuth_v1 "" "-" = { Authorization = "Basic Oi0=" }
 HttpClient.basicAuth_v1 ":" "" = { Authorization = "Basic Ojo=" }
 HttpClient.basicAuth_v1 "" ":" = { Authorization = "Basic Ojo=" }
 
-HttpClient.get_v5 "{ ] nonsense ^#( :" {} {} = Error "Couldn't resolve host name" // OCAMLONLY
-HttpClient.get_v5 "{ ] nonsense ^#( :" {} {} = Error "Invalid URI" // FSHARPONLY
-
 // -----------------------
 // Tests which should try to hit the actual internet
 // -----------------------
 
-// FSTODO: add these tests for all versions
```

Comment:
This FSTODO applies to the other httpclient tests below as well. (eg /just-a-path, ftp, file, etc). Can you implement the rest of the tests, or alternatively put back the FSTODO for now.

Reply:
Will implement shortly, thanks for reminder!